.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
```

Comment:
nit: "in" -> "using"

nit: "when <the> PR gets merged"

Reply:
what do you mean by "when PR gets merged"?

Reply:
Ooops, I wrote when __the__ PR gets merged, but I think the markdown engine swallowed my formatting.

Reply:
Note to self, markdown + HTML-style brackets for emphasis are not friends.